docs/DeveloperGuide.md

```diff
@@ -567,64 +567,211 @@ testers are expected to do more *exploratory* testing.
+`editPerson 1 n=Ryan`
 
-    1. Test case: `delete 0`<br>
-       Expected: No transaction is deleted. Error details shown in the status message. Status bar remains the same.
+Expected Output in the Person List Panel: The first person in the list has their name changed to "Ryan", and retains
+ the rest of their details.
 
-    1. Other incorrect delete commands to try: `deleteTransaction`, `deleteTransaction x`, `...` (where x is larger than the list size)<br>
-       Expected: Similar to previous.
+Expected Output in the Command Output Panel: Edit person message is shown along with the person's details.
 
-1. _{ more test cases …​ }_
+`editPerson 0`
 
-### Saving data
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
 
-1. Dealing with missing/corrupted data files
+### Listing multiple people with matching names
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is
+ "Roy Tan".
 
-1. _{ more test cases …​ }_
+`listPerson Roy`
+
+Expected Output in the Person List Panel: The person Roy Tan is displayed.
+
+Expected Output in the Command Output Panel: List person message is shown and states "1 person(s) listed".
+
+`listPerson Tan`
+
+Expected Output in the Person List Panel: Nothing is displayed.
+
+Expected Output in the Command Output Panel: List person message is shown and states "0 person(s) listed".
+
+### Settling an outstanding balance with a Person
+
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is "Alex" and their balance shown is +20.00.
+
+`settlePerson 1`
+
+Expected Output in the Transaction List Panel: A new transaction is created that is named "Settle balance with Alex".
+
+Expected Outp t in the Person List Panel: The balance of Alex is now "0.00"
+
+Expected Output in the Command Output Panel: Balance settled message is shown along with the name of Alex.
+
+### Sorting people by their balances
+
+Prerequisite: There are a few people stored in Spend N Split with varying balances.
+
+`sortPerson +`
+
+Expected Output in the Person List Panel: The people displayed are sorted in descending order of balance.
+
+Expected Output in the Command Output Panel: A message stating that the Person list is sorted in descending order of 
+balance is shown.
+
+`sortPerson -`
+
+Expected Output in the Person List Panel: The people displayed are sorted in ascending order of balance.
+
+Expected Output in the Command Output Panel: A message stating that the Person list is sorted in ascending order of
+balance is shown.
+
+### Adding a transaction
+
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is "Roy".
+
+`addTransaction d=bread n=Alex Yeoh c=25.00 n=Self w=1 n=Alex Yeoh w=1`
+
+Expected Output in the Transaction List Panel: New transaction is added into the list.
```

Comment:
Based on the pre-requisite, Alex Yeoh is not in the app so this should cause an error

Reply:
fixed. Thanks for the catch